upstairs/src/volume.rs

```diff
@@ -1551,6 +1666,9 @@ impl Volume {
                 old: o_sv_opts.target[cid],
                 new: n_sv_opts.target[cid],
             })
+        } else if read_only {
+            // VCRs are 100% the same
```

Comment:
Mabye: `VCRs are the same, ignoring generation numbers`

Reply:
agreed, added to comment in 15fb3b13